components/gif_picker/components/InfiniteScroll/index.tsx

```diff
@@ -0,0 +1,187 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {PureComponent} from 'react';
+
+type BaseElementProps = {
+    className?: string;
+}
+
+type InfiniteScrollDefaultProps = {
```

Comment:
I'm typing the default props this way because if I let TS infer the types with `typeof InfiniteScroll.defaultProps`, the props `containerHeight` and `scrollPosition` will be `never`.